op25/gr-op25_repeater/apps/rx.py

```diff
@@ -875,7 +875,7 @@ def process_ajax(self):
     def process_qmsg(self, msg):
         # return true = end top block
         RX_COMMANDS = 'skip lockout hold whitelist reload'
-        s = msg.to_string()
+        s = msg.to_string().decode('ascii')
```

Comment:
`utf-8` is the default encoding for Python 3 strings, any reason we need to use `ascii`?

Reply:
No overwhelming reason — I considered using `utf-8` and I think that would be fine also.

My reasoning is that since the messages being received are just a handful of fixed strings, this makes it clear that the messages *are* plain ASCII; if they were decoded as utf8 then it's not clear whether that's necessary.

(Either way, Python is just going to scan the bytes for validity and then copy them unmodified into the string object — the "decode" is trivial.)